tests/backup/backup_resiliency_test.go

```diff
@@ -1558,3 +1558,200 @@ var _ = Describe("{ScaleDownPxBackupPodWhileBackupAndRestoreIsInProgress}", func
+		})
+		Step("Applying label portworx.io/nobackup=true to worker nodes and verifying the license count", func() {
+			log.InfoD("Applying label portworx.io/nobackup=true to one of the worker node on source cluster and verifying the license count")
+			err := Inst().S.AddLabelOnNode(sourceClusterWorkerNodes[0], "portworx.io/nobackup", "true")
+			if err == nil {
+				sourceClusterWorkerNodeLabelApplied = true
+				sourceClusterWorkerNodeLabelRemoved = false
+			}
+			log.FailOnError(err, fmt.Sprintf("Failed to apply label portworx.io/nobackup=true to worker node %v", sourceClusterWorkerNodes[0].Name))
+			log.Infof("Sleeping for 10 minutes for license count to get updated")
+			time.Sleep(10 * time.Minute)
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			licenseCountUpdateCheck := func() (interface{}, bool, error) {
+				licenseInspectResponse, err = Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+				if err != nil {
+					return "", false, err
+				}
+				licenseResponseInfoFeatureInfo = licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()
+				if licenseResponseInfoFeatureInfo[0].Consumed == int64(len(totalNumberOfWorkerNodes)-1) {
+					return "", false, nil
+				}
+				return "", true, nil
+			}
+			_, err = task.DoRetryWithTimeout(licenseCountUpdateCheck, licenseCountUpdateTimeout, licenseCountUpdateRetryTime)
+			licenseInspectResponse, err = Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+			log.InfoD("License count after applying label portworx.io/nobackup=true to node %v is %v ", sourceClusterWorkerNodes[0].Name, licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()[0].Consumed)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying license count after applying label portworx.io/nobackup=true to node %v", sourceClusterWorkerNodes[0].Name))
+			log.InfoD("Switching cluster context to destination cluster")
+			SetDestinationKubeConfig()
+			log.InfoD("Applying label portworx.io/nobackup=true to one of the worker node on destination cluster and verifying the license count")
+			err = Inst().S.AddLabelOnNode(destinationClusterWorkerNodes[0], "portworx.io/nobackup", "true")
+			if err == nil {
+				destinationClusterWorkerNodeLabelApplied = true
+				destinationClusterWorkerNodeLabelRemoved = false
+			}
+			log.FailOnError(err, fmt.Sprintf("Failed to apply label portworx.io/nobackup=true to worker node %v", destinationClusterWorkerNodes[0].Name))
+			log.InfoD("Switching cluster context backup to source cluster")
+			err = SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+			log.Infof("Sleeping for 10 minutes for license count to get updated")
+			time.Sleep(10 * time.Minute)
```

Comment:
Why do you want sleep as well as timeout?

Reply:
Changed it:
1)removed 10 minutes sleep
2)Added timeout of 15 minuted with retry after every 1 min